packages/dx-react-grid/docs/reference/table-row-detail.md

```diff
@@ -63,7 +63,7 @@ tableBodyRows | Getter | Array&lt;[TableRow](table-view.md#table-row)&gt; | Body
 expandedRows | Getter | Array&lt;number &#124; string&gt; | Expanded rows
 getRowId | Getter | (row: [Row](grid.md#row)) => number &#124; string | The function used to get a unique row identifier
 setDetailRowExpanded | Action | ({ rowId }) => void | Expands the specified row
-tableViewCell | Template | { row: [TableRow](table-view.md#table-row), column: [TableColumn](table-view.md#table-column) } | A template that renders a table cell
+tableViewCell | Template | { row: [TableRow](table-view.md#table-row), column: [TableColumn](table-view.md#table-column), style?: Object, colspan: number } | A template that renders a table cell
```

Comment:
Is the `colspan` parameter always passed to the template or is it optional like `style`?